packages/plugin-eslint/src/lib/runner/lint.ts

```diff
@@ -16,7 +15,7 @@ export async function lint(
   const results = lintResults.map(
     (result): LintResult => ({
       ...result,
-      relativeFilePath: result.filePath.replace(process.cwd() + sep, ''),
+      relativeFilePath: toUnixPath(result.filePath, true),
```

Comment:
I think the usage would be more readable with the named options pattern:

```suggestion
      relativeFilePath: toUnixPath(result.filePath, { toRelative: true }),
```

Reply:
Done